### PR TITLE
enable passing ruleset dict to colin.run() and .get_checks

### LINKED ISSUE #98: colin API improvement: accept ruleset directly
This is coming from an existing test: https://github.com/user-cont/colin/pull/94#discussion-diff-183723728R54

Ideally colin would be able to accept ruleset config directly, without the need of creating files in filesystem.

### PR DESCRIPTION
**Pull request by TomasTomecek**
Fixes #98

### CODE REVIEW

File: colin/core/ruleset/ruleset.py [hunk 1]
```diff
@@ -60,6 +61,12 @@ def __init__(self, ruleset_name=None, ruleset_file=None):
 
                 logger.error(msg)
                 raise ColinRulesetException(msg)
+        elif ruleset:
+            self.ruleset_dict = ruleset
+        else:
+            logger.error("none of the arguments ruleset_{name,file}, ruleset was passed")
+            raise ColinRulesetException("No ruleset was selected.")
+        # TODO: validate ruleset
```
Comment (lachmanfrantisek) [inline colin/core/ruleset/ruleset.py]: We can leave this after the ruleset redesign. (And use jsonschema for that.)

**Comment by lachmanfrantisek** [comment]
Just two things:
- Tests failed in travis. I will look on it.
- Rebase? (...thanks..;-)

**Comment by TomasTomecek** [comment]
updated

I actually implemented https://github.com/user-cont/colin/pull/75#issuecomment-382623707 except that ./rulesets is last in the order

**Comment by lachmanfrantisek** [comment]
@TomasTomecek 
> I actually implemented #75 (comment) except that ./rulesets is last in the order
+1

**Comment by lachmanfrantisek** [comment]
> OK, I've looked at the Travis logs and there is problem with the default ruleset -- it is not used when no ruleset was specified.

Don't we want to use `default` ruleset by default? At least for CLI?

Now, `colin check something` fails:
```
$ colin check rhel
Error: No ruleset was selected.
```

**Comment by TomasTomecek** [comment]
Okay, I don't even know why I changed that.

**Comment by TomasTomecek** [comment]
should be fixed now

**Comment by lachmanfrantisek** [comment]
Thanks, I will use it in the #103

**Pull request merged**